components/brave_wallet_ui/page/screens/composer_ui/select_token_modal/select_token_modal.tsx

```diff
@@ -156,6 +156,55 @@ const getFullAssetBalance = (
   }, '0')
 }
 
+const getAccountsForSelectedAsset = (
+  selectedAsset: BraveWallet.BlockchainToken,
```

Comment:
Maybe rename and adjust props so it works for any object with `chainId` and `coin`? This would allow the logic to be reused for both assets and networks.

```suggestion
const getAccountsForNetwork = (
  network: Pick<BraveWallet.NetworkInfo, 'chainId' | 'coin'>,
```